myhpi/static/scss/myHPI.scss

```diff
@@ -5,20 +5,25 @@
 $primary: #F5631E;
 $secondary: #F5631E;
 $body-bg: #F5F5F5;
+$body-color-dark: white;
+$body-color: black;
+
+$gray-100: #e9e9e9;
+$gray-200: #EEE;
+$gray-300: #DDD;
+$gray-400: #CCC;
+$gray-500: #BBB;
+$gray-600: #999;
+$gray-700: #777;
+$gray-800: #555;
+$gray-900: #333;
 

```

Comment:
Is it wanted that there is no difference between the webpage background and the content background? The orange bar at the top and the footer at the bottom look a little bit lost to me without the content background

![grafik](https://github.com/user-attachments/assets/518c572d-d5f5-49be-b56f-7a2724ae4b16)

This might also fix the lack of contrast between table cells and content background:

![grafik](https://github.com/user-attachments/assets/8225ff7d-0923-4c90-ad1c-b2603c4f8f2a)